rules/CodeQuality/Rector/MethodCall/AssertTrueFalseToSpecificMethodRector.php

```diff
@@ -132,6 +133,13 @@ public function refactor(Node $node): ?Node
         return $node;
     }
 
+    private function resolveFirstArgument(FuncCall|Empty_ $firstArgumentValue): ?string
+    {
+        return $firstArgumentValue instanceof Empty_
+            ? 'empty'
```

Comment:
@TomasVotruba this is due to removal of EmptyNameResolver PR:

- https://github.com/rectorphp/rector-src/pull/4782

detected after rules-tests registered to phpunit.xml

https://github.com/rectorphp/rector-phpunit/pull/251#pullrequestreview-1618707993